Topten.RichTextKit/Topten.RichTextKit.csproj

```diff
@@ -37,7 +37,7 @@
 
   <ItemGroup>
     <PackageReference Include="SkiaSharp" Version="1.68.0" />
-    <PackageReference Include="SkiaSharp.HarfBuzz" Version="1.68.0" />
+    <PackageReference Include="HarfBuzzSharp" Version="2.6.1-rc.172" />
```

Comment:
2 things:
 - switch from SkiaSharp.HarfBuzz to HarfBuzzSharp so you don't have to depend on a "experimental"-type library.
 - Use the new version of harfbuzz with more features. _(I'll update when HarfBuzzSharp goes stable)_